lib/constants.rb

```diff
@@ -0,0 +1,23 @@
+module TicTacToe 
```

Comment:
`constants` is a super generic name. Remember that we always be thinking about keeping things modular. It makes more sense to create a separate file to store messaging as it may clutter the UI class than putting every single constant here. It's okay to keep constants in the class that's most relevant to the value.